### PR TITLE
Add link to 'data-platform' topic

### PR DESCRIPTION
**Pull request by taurus-forever**
Add a link to search all our repos quickly on Github.

### CODE REVIEW

**Review by deusebio — APPROVED** [review]
LGTM!

**Review by delgod — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
-Data Platform
+# Data Platform
 
 A repository to collect all of the helpers of Data Platform team at Canonical.
+
+A list of all the active repositories can be found using the topic '[data-platform](
+https://github.com/search?q=topic%3Adata-platform+org%3Acanonical+fork%3Atrue+archived%3Afalse&type=repositories)'.
```
Comment (delgod) [inline README.md:6]: maybe better link 
```suggestion
https://github.com/orgs/canonical/teams/data-platform-engineers/repositories)'.
```

**Review by delgod — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (taurus-forever) [reply README.md:6]: I have almost committed it... but tested in incognito window and the link cannot be publicly used (the repo is public).
It is a nice like to use internally, but maybe we should stay with "topics" link for public usage?

**Review by delgod — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (taurus-forever) [reply README.md:6]: Discussed with Mykola and agree to resolve this. Merging...

**Pull request merged**